HelloMaui/HelloMaui.csproj

```
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0-android;net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
```

Should we change this, so it still builds on Windows:
```suggestion
    <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('osx'))">net6.0-android;net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
    <TargetFrameworks Condition="!$([MSBuild]::IsOSPlatform('osx'))">net6.0-android;net6.0-ios</TargetFrameworks>
```